src/main/java/com/demo/controller/DemoApplicationController.java

```diff
@@ -14,6 +14,8 @@
 @RequiredArgsConstructor
```

Comment:
can we add a lombok `@SLf4j` annotation here for logging?

Reply:
Added @slf4j annotation as requested